test/Test.java

```diff
@@ -29,12 +29,13 @@ public static void main(String[] args) {
   static void checkAllOracleJDKs() {
     System.out.println();
     System.out.println("// oracle.com - latest");
-    checkOracleJDK("20", "latest");
+    checkOracleJDK("21", "latest");
     checkOracleJDK("17", "latest");
 
     System.out.println();
     System.out.println("// oracle.com - archive");
-    Stream.of("19", "19.0.2", "19.0.2").forEach(version -> checkOracleJDK("19", version));
+    Stream.of("20").forEach(version -> checkOracleJDK("20", version));
```

Comment:
Just taking inspiration from the lines below, should this also cover patch versions of 20?

```suggestion
    Stream.of("20", "20.0.1", "20.0.2").forEach(version -> checkOracleJDK("20", version));
```